src/wallet/collector.rs

```diff
@@ -114,7 +113,6 @@ impl Collector {
                 .arg("--url")
                 .arg(&self.api_uri)
                 .arg("wallet")
-                .arg("--wait-for-sync")
```

Comment:
The `--ckb-indexer-url` argument is also needed.

https://github.com/nervosnetwork/ckb-cli/blob/e41c4a35d24987fd3ee43887929967be19ffc0b9/src/main.rs#L381-L389

Reply:
Didn't notice this because I have run ckb-cli interactively before and there are saved urls.

This argument will need to be added to `capsule deploy` too and passed to ckb-cli?

Reply:
Yes, just like the argument `--url` above.